site/en/docs/extensions/mv3/tut_websockets/index.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Using WebSockets in Service Workers'
```

Comment:
I feel like "extensions" should be mentioned in the title

Reply:
It's in the seoTitle, not sure we need it in the normal title as well.